db/facade.py

```diff
@@ -0,0 +1,67 @@
+"""Database Facade."""
+# import boto3
+from model.user import User
+
+
+class DBFacade:
+    """
+    A database facade that gives an overall API for any databases.
+
+    Currently, we plan on having DynamoDB, but other databases, such as MongoDB
+    or Postgres are also being considered.
+    """
+
+    def __init__(self):
+        """Initialize facade using DynamoDB settings (for now)."""
+        # TODO change this to production and not localhost
+        # self.ddb = boto3.resource("dynamodb", region_name="",
+        #                          endpoint_url="http://localhost:8000")
+        pass
+
+    def __str__(self):
+        """Return a string representing this class."""
+        return "Database Facade"
+
+    def store_user(self, user):
+        """
+        Store user into users table.
+
+        :param user: A user model to store
+        """
+        # Assume that the tables are already set up this way
```

Comment:
Keep in mind that eventually we'll want some error handling to check this... that should probably be handled by the DynamoDb-specific portion of the code, though.